test/0.8.9/oracle/hash-consensus-frames.js

```diff
@@ -161,7 +236,7 @@ contract('HashConsensus', ([admin, member1, member2]) => {
     })
 
     it(`decreasing the frame size cannot decrease the current reference slot`, async () => {
-      assert.equal(+await consensus.getTime(), computeTimestampAtEpoch(1))
+      assert.equal(+(await consensus.getTime()), computeTimestampAtEpoch(1))
```

Comment:
could we change prettier settings to not add parentheses around await in cases like this one? it's much less readable this way imo

Reply:
not critical, a nitpick

Reply:
Faced same issues with prettier in different PR. My prettier is pulling config from root of the repo but a lot of other files are not formatted at all. In a different PR, we can update config and format all(or just currently relevant) files.

Reply:
let's do it later in a single batch I suggest